BEE-44920 - Fixed an error that prevented to run UpdateApp procedure. (#312)
electric-cloud/EC-WebSphere: help/help.adoc

[[rns]]
== Release notes

=== EC-WebSphere 3.0.1

> - Fixed an issue that prevented to run UpdateApp procedure.

Reviewer: ```suggestion
- Fixed an issue that prevented the *UpdateApp* procedure from running.
```

Author: Updated in last commit
